### DISCUSSION TITLE
Light mode as default

### DISCUSSION THREAD

**Discussion by rafaeldjpbrochado** [iotaledger/iota-wiki#189] [General]
Given that most of our graphics are designed for light mode, should the wiki be light mode by default?

With dark mode, you see these kinds of contrasts, which don't look as nice:

![image](https://user-images.githubusercontent.com/39466455/132469471-da7a9848-3d52-4fcf-a3fd-1c9c7426c92f.png)

Of course, to mitigate this problem we could also just try to make dark mode variations of all these graphics, but that could be a bit overkill.

**Comment by Dr-Electron** [comment]
I think we can lock this discussion. We switched to dark mode images everywhere 😆
